sql/views.sql

```diff
@@ -55,12 +55,14 @@ CREATE OR REPLACE VIEW timescaledb_information.job_stats as
                    WHEN js.last_run_success = 'f' THEN 'Failed'
               END
     END as last_run_status,
-    CASE WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
+    CASE WHEN j.scheduled = false THEN 'Paused'
+         WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
```

Comment:
For clarity, maybe add (although not strictly necessary):
```suggestion
         WHEN j.scheduled AND js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
```

This also makes this line independent of other cases (e.g., in case other cases change).